app/services/degrees/create_from_hesa.rb

```diff
@@ -18,16 +18,19 @@ def call
     attr_reader :hesa_degrees, :trainee
 
     def create_degrees!
-      hesa_degrees.each do |hesa_degree|
-        degree = trainee.degrees.find_or_initialize_by(subject: hesa_degree[:subject])
+      hesa_degrees.each_with_object([]) do |hesa_degree, array|
```

Comment:
Not a biggy but I think we could just `.map` here and not have to handle appending to the array.